rest_framework/templatetags/rest_framework.py

```diff
@@ -164,6 +164,12 @@ def format_value(value):
 
 
 @register.filter
+def items(value):
+    if hasattr(value, 'items'):
+        return value.items()
```

Comment:
Reckon we should probably omit the `if` clause here.
Also worth adding a docstring mentioning why we need this.